src/Fantomas.Core/FormatConfig.fs

```diff
@@ -227,7 +227,7 @@ type FormatConfig =
 
       [<Category("Convention")>]
       [<DisplayName("Applies the Stroustrup style to the final (two) array or list argument(s) in a function application")>]
```

Comment:
Not sure either way, but should we change anything about this displayName to mention elmish or do you think it's fine to stay as is??

Reply:
I think this is fine for now. We can revisit this later.
Little secret, but somebody is actually doing anything with these attributes anyway.
We expose them via `Fantomas.Client` so IDEs can request them but none of them has implemented this.